src/Core/Application/Application.csproj

```diff
@@ -10,9 +10,13 @@
         <PackageReference Include="MediatR.Extensions.Microsoft.DependencyInjection" Version="9.0.0" />
         <PackageReference Include="Microsoft.Extensions.Caching.Abstractions" Version="6.0.0" />
         <PackageReference Include="Microsoft.Extensions.Localization" Version="6.0.0" />
+        <PackageReference Include="Microsoft.EntityFrameworkCore" Version="6.0.0" />
```

Comment:
The whole point of Repository Pattern is to remove the Dependency with EFCore or any other ORMs in the Application / Core layer.

Reply:
Yes, the dependency has been removed.